### PR TITLE
Adding listen address

### PR DESCRIPTION
**Pull request by jstoja**
You can specify the port, but not the listening address. It's possible now.

### CODE REVIEW

**Comment by jstoja** [comment]
@Gegonz ? :)

**Comment by Gegonz** [comment]
This PR only modify print.
TODO: update this PR with https://gstreamer.freedesktop.org/data/doc/gstreamer/head/gst-rtsp-server/html/GstRTSPServer.html#gst-rtsp-server-set-address

**Comment by jstoja** [comment]
Indeed, changing that immediately.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 4. `cd build`
 5. `cmake ..`
 6. `make`
-7. `Usage: ./camera_emulation_server [-a address] [-b port] [-r route] [-i input] [-u username] [-p password] [-s 'width'x'height'] [-f framerate]`
+7. `Usage: ./camera_emulation_server [-a address] [-l address] [-b port] [-r route] [-i input] [-u username] [-p password] [-s 'width'x'height'] [-f framerate]`
```
Comment (Gegonz) [inline README.md]: Duplicate, you can remove -a

Comment (jstoja) [inline README.md]: Fixed !

**Comment by Ullaakut** [comment]
Is this PR abandoned, in progress or can it be merged?

**Comment by Gegonz** [comment]
You can merge it

**Pull request merged**